utils/generate-command-help.rb

```diff
@@ -1,131 +0,0 @@
-#!/usr/bin/env ruby
```

Comment:
we now wanna keep this file

Reply:
Oh, I thought the plan was to keep `help.h` constant. Will fix.